packages/desktop/src/renderer/components/widgets/channels/ChannelInput/utils/emojiCodes.ts

```diff
@@ -671,6 +896,19 @@ const emoticons: EmojiMapping = {
   ":'-(": '😢',
   ":'D": '😂',
   ":'-)": '😂',
+  ':3': '😊',
+  ':-3': '😊',
+  XD: '😆',
```

Comment:
```suggestion
  'XD': '😆',
```
Is there a reason this wasn't wrapped in quotes?